### PR TITLE
Further fixes

### PR DESCRIPTION
**Pull request by ltalirz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,50 +1,55 @@
 # sssp-protocol
-This repository contains all the necessary workflows and tools in order to run the SSSP (standard solid state pseudopotential) protocol for testing pseudopotentials.
+This repository contains all the necessary workflows and tools in order to run
```
Comment (CasperWA) [inline README.md]: For git reasons, it's nicer to have complete single sentences on a single line (as it was).

**Comment by ltalirz** [comment]
By the way guys @CasperWA @giprandini 
any news on this? what is the current state?

**Comment by CasperWA** [comment]
> any news on this? what is the current state?

I have a pip freeze list and an "epfl_theos" folder full of workflows from @AntimoMarrazzo.
I am currently setting up a virtualenv on my machine to try and reproduce this enviroment and have the workflows running (using AiiDA v0.7.0.1, with some minor fixes).

File: README.md [hunk 2]
```diff
@@ -7,44 +7,47 @@ This repository contains all the necessary workflows and tools in order to run t
...
 ```
-
+ 
 - Compile the Fortran77 subroutine for the Fermi energy with `efermi_f2py_make.sh` through f2py
 
 #### Usage example
 
-We include a simple example to test the SsspWorkflow on elemental palladium and PAW pseudopotential from the PsLibrary 0.3.1. 
-In order to run the example you need to:
+This demonstrates how to run the SsspWorkflow on elemental palladium for a PAW
```
Comment (CasperWA) [inline README.md]:
```suggestion
This demonstrates how to run the SsspWorkflow on elemental palladium for a PAW pseudopotential from the PsLibrary 0.3.1.
```

And lose the next line.

**Comment by ltalirz** [comment]
Ok, great - keep me updated!

**Comment by ltalirz** [comment]
By the way, @giprandini this is ready to "squash & merge"

**Review by CasperWA — APPROVED** [review]
:+1:

**Pull request merged**